src/libraries/System.Private.CoreLib/src/System/Numerics/IFloatingPointIeee754.cs

```diff
@@ -102,6 +102,34 @@ public interface IFloatingPointIeee754<TSelf>
         /// <returns><c>true</c> if <paramref name="value" /> is subnormal; otherwise, <c>false</c>.</returns>
         static abstract bool IsSubnormal(TSelf value);
 
+        /// <summary>Compares two values to compute which is greater.</summary>
+        /// <param name="x">The value to compare with <paramref name="y" />.</param>
+        /// <param name="y">The value to compare with <paramref name="x" />.</param>
+        /// <returns><paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
+        /// <remarks>For <see cref="IFloatingPointIeee754{TSelf}" /> this method matches the IEEE 754:2019 <c>maximumMagnitudeNumber</c> function. This requires NaN inputs to not be propagated back to the caller and for <c>-0.0</c> to be treated as less than <c>+0.0</c>.</remarks>
+        static abstract TSelf MaxMagnitudeNumber(TSelf x, TSelf y);
+
+        /// <summary>Compares two values to compute which is greater.</summary>
+        /// <param name="x">The value to compare with <paramref name="y" />.</param>
+        /// <param name="y">The value to compare with <paramref name="x" />.</param>
+        /// <returns><paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
+        /// <remarks>For <see cref="IFloatingPoint{TSelf}" /> this method matches the IEEE 754:2019 <c>maximumNumber</c> function. This requires NaN inputs to not be propagated back to the caller and for <c>-0.0</c> to be treated as less than <c>+0.0</c>.</remarks>
+        static abstract TSelf MaxNumber(TSelf x, TSelf y);
```

Comment:
The docs on this are identical to the docs on MaxMagnitudeNumber, with the sole exception of saying that it matches maximumNumber instead of maximumMagnitudeNumber, but for that to be meaningful to me I need to know what IEEE says those are :)  Can we be more explicit in the summary about the comparison being performed in order to differentiate these?

Same for the comments below.

Reply:
It also differs on `This requires NaN inputs to be propagated` vs `This requires NaN inputs to not be propagated`

I'll submit a follow up PR to update the docs to be more explicit in the summary.

Reply:
Actually, I'll update them in this PR. This isn't going to make the snap since `op_Modulus` is still failing in CI only

Reply:
They both say "This requires NaN inputs to not be propagated back to the caller".  Is that just a typo then?

Reply:
Oh sorry, I misread this as saying `MaxNumber` and `Max` docs were identical.

I did fix it up so `Max`, `MaxNumber`, and `MaxMagnitudeNumber` all differ in their docs

Reply:
Basically:
* `Max` returns the greater of `x` vs `y`, preferring `NaN` when present
  * `Max(2, -3)` returns `2`
  * `Max(2, NaN)` returns `NaN`
* `MaxMagnitude` returns the greater of `x` or `y` comparing using their absolute value (magnitude), preferring `NaN` when present
  * `MaxMagnitude(2, -3)` returns `-3`
  * `MaxMagnitude(2, NaN)` returns `NaN`
* `MaxNumber` returns the greater of `x` vs `y`, not preferring `NaN` when present
  * `MaxNumber(2, -3)` returns `2`
  * `MaxNumber(2, NaN)` returns `2`
* `MaxMagnitudeNumber` returns the greater of `x` or `y` comparing using their absolute value (magnitude), not preferring `NaN` when present
  * `MaxMagnitudeNumber(2, -3)` returns `-3`
  * `MaxMagnitudeNumber(2, NaN)` returns `2`